scripts/zones/Quicksand_Caves/npcs/amk_qm.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
+-- Area: Quicksand Caves
+-- NPC: qm (???)
+-- Mission: AMK - Rescue! A Moogle's Labor of Love
```

Comment:
Probably worth adding a comment saying this script is shared between all the AMK qm's for this mission, to avoid confusion or someone bloating out the zone with multiple empty files - one for each qm